tests/main/selinux-data-context/task.yaml

```diff
@@ -0,0 +1,36 @@
+summary: Check that SELinux file context transitions work
+
+description: |
+    Verify that automatic file context transitions work and cover typical use cases
+
+# TODO: enable centos once we are able to build the policy
+systems: [fedora-*, centos-*]
```

Comment:
We should be able to build the policy for CentOS now.

Reply:
Yup, waiting for @sergiocazzolato to greenlight CentOS 7.6. There was some hiccup with `golang` packaging that made building `snapd` fail, but `golang-1.11.2-3.el7` is in EPEL now, so we should be fine.